app/buyers/views/buyers.py

```diff
@@ -246,3 +249,54 @@ def delete_a_brief(framework_slug, lot_slug, brief_id):
             url_for('.view_brief_summary', framework_slug=framework_slug, lot_slug=lot_slug,
                     brief_id=brief_id, delete_requested=True)
         )
+
+
+@buyers.route("/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/answer-question",
```

Comment:
Seems like we might not need `framework_slug` and `lot_slug` for this route - `brief_id` is the only thing that really matters for adding clarifications.  But given that all the other routes here follow the same pattern maybe we should stick with it.  I'm not sure what's best... for the supplier responses to a brief we went with just including the `brief_id`.